outbound/508-checklist.md

```diff
@@ -0,0 +1,48 @@
+| 8 | [Creating Links with Unique and Descriptive Names](https://www.section508.gov/training/presentations/aed-cop-pptx08/) | <ul><li>Ensure all links have a unique URL or a unique descriptive name.</li><li>Avoid descriptions like “Link” or “Click here.”</li><li>By adding transparent text boxes with links to the front of a background image. Ensures hyperlinks rather than just visible URLs.</li><li>When possible avoid using images with embedded text.</li><li>Link descriptions can be added to the “Alt Text” feature.</li></ul> | |
+| 9 | [Making Vital Background Information Accessible](https://www.section508.gov/training/presentations/aed-cop-pptx09/) | <ul><li>Make sure key details are clearly stated within the slide itself, using techniques like adding headers/footers, utilizing clear and concise text, providing alt text for images, maintaining sufficient color contrast, and checking the reading order to guarantee information is presented logically for screen readers.</li></ul> | |
+| 10 | [Formatting Text for the Intended Language](https://www.section508.gov/training/presentations/aed-cop-pptx10/) | <ul><li>For presentations using multiple languages, use the language tool to programmatically set the presentation language to enable assistive technology to infer and correctly pronounce content.</li></ul> | |
+| 11 | [Ensuring Descriptions of Embedded Audio, Video and Multimedia Files are Accurate](https://www.section508.gov/training/presentations/aed-cop-pptx11/) | <ul><li>Carefully name your media files with descriptive titles that clearly convey the content, and when necessary, add detailed captions or alternative text to explain what is being shown or heard.</li><li>Include key details like speaker names, topic, or location when relevant.</li><li>When adding images with embedded audio or video, provide alternative text that describes what is visually happening in the image alongside the audio or video content.</li></ul> | |
+| 12 | [Excluding Flashing Objects](https://www.section508.gov/training/presentations/aed-cop-pptx12/) | <ul><li>Remove flashing objects from slides since they can cause seizures</li</ul>
+| 13 | [Saving in the .pptx Format with a Descriptive Filename](https://www.section508.gov/training/presentations/aed-cop-pptx13/) | <ul><li>Use a descriptive file name to identify the document and its purpose so everyone can locate, open, and switch between documents.</li></ul> | |
```

Comment:
Something that we often do is

| 14 | [Export the .pptx to a .pdf](https://www.section508.gov/create/pdfs/) | <ul><li>PDF files typically look the same on every device, and don't rely on dynamically loading things like typefaces or images from local systems. PDFs help preserve formatting, and limit usage of features like animations or transitions, that can be difficult to process in screenreaders or other accessibility tools.</li></ul> | |

Reply:
14 added